### PR TITLE
Add compilation options for datatransform dependencies in dlr_relayvm.cc

### PR DESCRIPTION
**Pull request by yunma10**
-Fix Typo in CMakeList.txt
-Compilation options for datatransform dependencies in dlr_relayvm.cc

### CODE REVIEW

**Review by minlu1021 — APPROVED** [review]

**Review by trevor-m — CHANGES_REQUESTED** [review]
Thanks @yunma10 

Could you enable data transform in CI build so that the tests are still ran.
https://github.com/neo-ai/neo-ai-dlr/blob/main/Jenkinsfile#L94

File: CMakeLists.txt [hunk 1]
```diff
@@ -233,6 +233,11 @@ if(WITH_HEXAGON)
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DDLR_HEXAGON")
     list(APPEND DLR_SRC "src/dlr_hexagon/dlr_hexagon.cc")
 endif()
+
+if(ENABLE_DATATRANSFORM)
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DENABLE_DATATRANSFORM")
```
Comment (trevor-m) [inline CMakeLists.txt:238]: Could you use `add_definitions(-DENABLE_DATATRANSFORM)` instead of CXX flags?

**Review by trevor-m — CHANGES_REQUESTED** [review]

File: CMakeLists.txt [hunk 2]
```diff
@@ -307,11 +312,13 @@ if(NOT(AAR_BUILD))
...
   if(NOT(ENABLE_DATATRANSFORM))
-    list(REMOVE_ITEM TEST_SRCS ${CMAKE_CURRENT_SOURCE_DIR}/tests/cpp/dlr_data_transform_test.cc)
+    list(REMOVE_ITEM TEST_SRCS 
+        ${CMAKE_CURRENT_SOURCE_DIR}/tests/cpp/dlr_data_transform_test.cc
+        ${CMAKE_CURRENT_SOURCE_DIR}/tests/cpp/dlr_pipeline_skl_xgb_test.cc)
   endif()
 
   if(WITH_HEXAGON)
-    file(GLOB HEXAGON_TEST_SRCS tests/cpp/dlr_hexagon/*.cc)
+    file(GLOB HEXAGON_TEST_SRS tests/cpp/dlr_hexagon/*.cc)
```
Comment (trevor-m) [inline CMakeLists.txt:321]: Typo SRS -> SRCS?

**Review by trevor-m — CHANGES_REQUESTED** [review]

File: CMakeLists.txt [hunk 3]
```diff
@@ -484,3 +491,4 @@ if(NOT(AAR_BUILD))
 endif()
 # Group sources
 #auto_source_group("${SOURCES}")
+message(${CMAKE_CXX_FLAGS})
```
Comment (trevor-m) [inline CMakeLists.txt:494]: Remove?

**Review by trevor-m — CHANGES_REQUESTED** [review]

**Review by trevor-m — APPROVED** [review]

**Pull request merged**